os/kernel/log_dump/log_dump_manager_start.c

```diff
@@ -0,0 +1,52 @@
+/****************************************************************************
+ *
+ * Copyright 2023 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <tinyara/log_dump/log_dump_internal.h>
+#include <tinyara/log_dump/log_dump.h>
+
+/****************************************************************************
+ * Pre-processor Definitions
+ ****************************************************************************/
+/****************************************************************************
+ * Private Type Declarations
+ ****************************************************************************/
+/****************************************************************************
+ * Public Variables
+ ****************************************************************************/
+/****************************************************************************
+ * Private Variables
+ ****************************************************************************/
+/****************************************************************************
+ * Private Functions
+ ****************************************************************************/
+/****************************************************************************
+ * Public Functions
+ ****************************************************************************/
+
+void log_dump_manager_start(void) {
+	int pid;
+	svdbg("Starting log_dump manager thread\n");
```

Comment:
How about printing this log after checking the successful thread creation?
If the kernel thread is not created well, both `Starting` and `Failed` are showing, so can be confused.